ui/app/src/App.tsx

```diff
@@ -55,6 +60,28 @@ function App() {
 
   const [stream, setStream] = useState<Stream | null>(null);
 
+  // Get streamManager to pass it as props to cancelstream
+  const sm = new StreamManager(
+    config.streamManagerAddress as `0x${string}`,
```

Comment:
question: Does `0x${string}` work?  Don't think I've seen this one before.  If you know what it's called, lmk.

Reply:
<img width="432" alt="Screenshot 2023-10-17 at 09 25 51" src="https://github.com/ApeWorX/ApePay/assets/117547827/e368520f-437e-4fcb-baca-8485e50d0aac">

This is an alias from Viem but you're right; I'll use Address directly 👍

Reply:
Syntax just through me off.  Looked it up, it's [template literal types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html).  Neat.